e2e/tests-dfx/update_settings.bash

```diff
@@ -42,6 +42,20 @@ teardown() {
   assert_match "Freezing threshold: 100_000_000_000"
 }
 
+@test "set wasm memory limit" {
```

Comment:
What about expanding this test to also cover following paths?
* `dfx canister create --wasm-memory-limit`
* Setting the field in `dfx.json`